test/e2e_test.go

```diff
@@ -155,19 +162,19 @@ func TestRekor(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			ctx := logtesting.TestContextWithLogger(t)
 			c, ns, cleanup := setup(ctx, t, setupOpts{})
-			defer cleanup()
+			t.Cleanup(cleanup)
 
 			// Setup the right config.
 			resetConfig := setConfigMap(ctx, t, c, test.cm)
-			defer resetConfig()
+			t.Cleanup(resetConfig)
 
 			createdObj := tekton.CreateObject(t, ctx, c.PipelineClient, test.getObject(ns))
 
 			// Give it a minute to complete.
-			waitForCondition(ctx, t, c.PipelineClient, createdObj, done, 60*time.Second)
+			waitForCondition(ctx, t, c.PipelineClient, createdObj, done, time.Minute)
```

Comment:
do these two lines also need nil checks? (here and below)

Reply:
cleaned this up everywhere